src/lib/es2015.proxy.d.ts

```diff
@@ -1,18 +1,17 @@
 interface ProxyHandler<T extends object> {
     getPrototypeOf? (target: T): object | null;
-    setPrototypeOf? (target: T, v: any): boolean;
+    setPrototypeOf? (target: T, v: object | null): boolean;
     isExtensible? (target: T): boolean;
     preventExtensions? (target: T): boolean;
-    getOwnPropertyDescriptor? (target: T, p: PropertyKey): PropertyDescriptor | undefined;
-    has? (target: T, p: PropertyKey): boolean;
-    get? (target: T, p: PropertyKey, receiver: any): any;
-    set? (target: T, p: PropertyKey, value: any, receiver: any): boolean;
-    deleteProperty? (target: T, p: PropertyKey): boolean;
-    defineProperty? (target: T, p: PropertyKey, attributes: PropertyDescriptor): boolean;
-    enumerate? (target: T): PropertyKey[];

```

Comment:
The&nbsp;`enumerate(…)`&nbsp;method was&nbsp;removed in&nbsp;**ES2016** from&nbsp;both&nbsp;`Reflect` and&nbsp;`ProxyHandler`s, due&nbsp;to&nbsp;implementers deciding&nbsp;against&nbsp;it.